### PR TITLE
Update with content changes

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates the product details to match the [copydoc](https://docs.google.com/document/d/1FeFQOztvyxnXP4Gp5nsTe0iV7mMNtY4PSfiKxvuPgWI/edit#heading=h.gyritmbc0ou4). It basically combines the 'about' and 'resources' lists into one singular 'about' list
- Updates the JS only handle a singular 'About' list.
- Some design changes (mostly with alignment) based on [the figma](https://www.figma.com/file/BSP7FqnRMcMvO7kCozJKUa/Ubuntu.com-navigation-update-Iteration-9?type=design&node-id=6342-3464&mode=design&t=QQ3MqZ54tguUFufH-0) design

### CODE REVIEW

**Comment by petesfrench** [comment]
@juanruitina Could you take a look at this please?

**Comment by juanruitina** [comment]
One mistake on my side: please change the "Contact us" link to https://canonical.com/contact-us

Other than that, changes look good to me.

File: src/js/product-details.js [hunk 1]
```diff
@@ -132,71 +132,37 @@ const canonicalProducts = {
...
+      url: 'https://ubuntu.com/engage',
     },
     {
       title: 'Partners',
       url: 'https://canonical.com/partners',
     },
     {
-      title: 'Contact',
+      title: 'Contact us',
       url: 'https://ubuntu.com/contact-us',
```
Comment (mtruj013) [inline src/js/product-details.js:166]:
```suggestion
      url: 'https://canonical.com/contact-us',
```

Comment (mtruj013) [reply src/js/product-details.js:166]: Should route here according to doc

**Review by mtruj013 — APPROVED** [review]
LGTM with one small comment

**Pull request merged**